saleor/graphql/checkout/mutations.py

```diff
@@ -127,21 +127,23 @@ def clean_input(cls, info, instance, input):
             default_billing_address = user.default_billing_address
             default_shipping_address = user.default_shipping_address
 
-        if 'shipping_address' in input:
-            shipping_address = cls.validate_address(input['shipping_address'])
+        if 'shipping_address' in raw_input:
```

Comment:
I know that we are py3 but it seems that GH highlights `raw_input` same as `input`. Maybe we should use a different name? 
@akjanik, @NyanKiyoshi What do you think?

Reply:
Maybe `data`? Django forms use such name to initialize form instances.

Reply:
`data` would be good!